test/ably/resttoken_test.py

```diff
@@ -179,6 +180,12 @@ def test_toke_details_from_json(self):
             TokenDetails.from_json(token_details_str),
         )
 
+    # Issue #71
+    @dont_vary_protocol
+    def test_request_token_float(self):
+        lifetime = datetime.timedelta(hours=4)
+        self.ably.auth.request_token({'ttl': lifetime.total_seconds() * 1000})
```

Comment:
In general milliseconds are expected, without an idiomatic variant.
The exceptions are:
- httpOpenTimeout, httpRequestTimeout and httpMaxRetryDuration are interpreted as seconds
- The start/end Time parameters found in several places can be either milliseconds from the epoch, or a datetime object.

If you want I could make a different PR after this one, where a timedelta object is accepted for Duration values, and a datetime object is accepted for Time values. Don't know if changing how an integer is interpreted (ms or s) is worth it, as it would break backwards compatibility.